CrossrefPlugin.php

```diff
@@ -418,46 +419,54 @@ public function getAllowedDoiTypes(): array
     /**
      * Make additional validation checks against publishing requirements
      *
+     * @throws Exception
      * @see PKPPublicationService::validatePublish()
      */
     public function validate(string $hookName, array $args): bool
     {
-        $errors =& $args[0];
+        $errors = & $args[0];
         $submission = $args[2];
         $context = Application::getContextDAO()->getById($submission->getData('contextId'));
-        $publication = $submission->getCurrentPublication();
+        $publication = $submission->getCurrentPublication(); /* @var $publication Publication */
         $issueId = $publication->getData('issueId');
         $rules = [
             'onlineIssn' => ['required_without:printIssn', 'string'],
-            'doi' => ['required', 'url'],
+            'doi' => ['required', 'string'],
             'issueId' => ['required', 'integer'],
```

Comment:
@kaitlinnewson regarding to this rule , we are making the issue association non required as part of pkp/pkp-lib#9295 . so wondering is this a crossref requirement to must have an issue for deposition . AFAIK, corssref does not have such strict requirement and in that case we need relax this validation rule . can you confirm ?

Reply:
Hi @touhidurabir, you are correct, Crossref does not require an issue in their schema so this can be relaxed.